core/vm/actor/builtin/v0/miner/miner_actor.hpp

```diff
@@ -335,4 +336,19 @@ namespace fc::vm::actor::builtin::v0::miner {
 
   extern const ActorExports exports;
 
+  /**
+   * Collects and stores precommit messages to make
+   * a packaged sending of a several messages within one transaction
+   * which reduces the general amount of transactions in the network
+   * with reduction of a gas fee for transactions.
+   */
+
+  struct PreCommitBatch : ActorMethodBase<25> {
```

Comment:
Mikhail is right, the method was introduced in v5 actor. Please, create file `core/vm/actor/builtin/v5/miner/miner_actor.hpp' and move `PreCommitBatch` method to the file. Don't foget include the file.

Reply:
We decided to not create v5/miner/miner_actor.hpp
@Elestrias, just provide 25 instead of PreCommitBatch::Number and put the comment
`// TODO tagirov Miner actor v5 PreCommitBatch`

Reply:
And remove all changes from current file

Reply:
So, I missed that we need Params and CBOR for them. We need to create v5 file and define it